Informer/Program.cs

```diff
@@ -7,8 +7,19 @@
 using Informer.Repository.Sqlite.Repositories;
 using Microsoft.EntityFrameworkCore;
 
+var MyAllowSpecificOrigins = "_myAllowSpecificOrigins";
```

Comment:
why do we need this? even if we do need this, it has a bad name. MyBlahBlahBlah isn't a good way to name an identifier